playwright/tests/office/ppms/happyPathFlows.spec.js

```
@@ -25,7 +27,7 @@ test.describe('Services counselor user', () => {
     await ppmPage.waitForLoading();
 
     // Verify SIT info
-    await expect(page.getByText('Government constructed cost: $326')).toBeVisible();
+    await expect(page.getByText('Government constructed cost:')).toBeVisible();
```

👍 Love this. Thanks. We have so many hardcoded values that when something minor changes the tests fail.